lib/NewDataList/NewDataList.js

```diff
@@ -82,7 +82,7 @@ module.exports = kind({
 		// potentially not in view, and focusing would cause it to
 		// scroll it into view for no reason apparent to the user.
 		// Instead, we just focus the first visible list element.
-		if (event.focusType === 'default' && event.index) {
+		if (event.focusType === 'default' && typeof event.index === 'number') {
```

Comment:
In other places, we've performed this kind of check:

```
(event.index || event.index === 0)
```

Curious if there's a perf difference between that and `typeof`. I'd try with jsperf but, alas, it's still down. Any ideas?